app/src/main/res/values/strings.xml

```diff
@@ -235,7 +235,29 @@
     <string name="firstrun_close_button">OK, got it!</string>
     <string name="firstrun_skip_button">Skip</string>
     <string name="firstrun_next_button">Next</string>
+   
+    <!-- The welcome titles in Firefox Lite 2.5 -->
+    <string name="firstrun_fxlite_2.5_title_1">Hello!</string>
+    <!-- The welcome titles in Firefox Lite 2.5 for asking users to choose a main purpose of downloading Firefox Lite -->
+    <string name="firstrun_fxlite_2.5_title_2">What brings you to Firefox Lite?</string>
+    <!-- 4 content themes for users to select in order to get the most relevant content on Firefox Lite Home -->
+    <string name="content_theme_1">Browse the Web</string>
+    <string name="content_theme_2">Find Good Deals</string>
+    <string name="content_theme_3">Play Games and Videos</string>
+    <string name="content_theme_4">Read Trending News</string>
+    <!-- Button to finish setting -->
+    <string name="button_done">DONE</string>
```

Comment:
It's not necessary to have all-caps string for the button text. (Just define `Done`, we can make it all-caps for button)

Reply:
OK